documentation/memory/step2/using_heap_snapshot.md

```diff
@@ -0,0 +1,9 @@
+# Using Heap Snapshit
```

Comment:
```suggestion
# Using Heap Snapshot
```

Reply:
@cjihrig @hekike - I made this change while landing myself, hope you are good with that!